app/models/article.js

```diff
@@ -21,4 +27,9 @@ const articleSchema = new mongoose.Schema({
     ]
 })
 
+function categoryValidation(val) {
+    const categories = JSON.parse(require('fs').readFileSync(require('path').join(__dirname, '../staticdata/categories.json'), 'utf-8'))
```

Comment:
I believe you can require JSON files in NodeJS. Like `require('./someJSON.json')`.